### ISSUE TITLE
yo can you somehow make it so we dont need to keep our pc on like something like replit or a vps idk

### ISSUE THREAD

**Issue by Robthekilla** [machacks02/dankMemerHack#9]
the title says it

**Comment by nostorian** [comment]
it is better you don't run automated programs like this for a long time
keep taking breaks every 30 minutes or so...
if you keep the selfbot on 24/7 by replit or any vps, by the time you get back you would be banned from dank memer's automod as it would detect some kinda selfbot running coz no human would send commands 24/7.

and in worst case, if discord spam filter catches you, your account will get disabled too without any appeal.

**Comment by TG-KRISH** [comment]
dude, make a private server and use in it dank memer has no server analytics yet

**Comment by CattopyTheWeb** [comment]
> it is better you don't run automated programs like this for a long time keep taking breaks every 30 minutes or so... if you keep the selfbot on 24/7 by replit or any vps, by the time you get back you would be banned from dank memer's automod as it would detect some kinda selfbot running coz no human would send commands 24/7.
> 
> and in worst case, if discord spam filter catches you, your account will get disabled too without any appeal.

yes you are right, self-botting a user account is against discord TOS
